src/components/input/NumericInput.tsx

```diff
@@ -220,7 +226,7 @@ const NumericInput: GenericComponent<NumericInputProps> = forwardRef<HTMLElement
       if (stepper === 'connected') {
         return (
           <Button
-            disabled={isMaxReached()}
+            disabled={typeof increaseDisabled !== 'undefined' ? increaseDisabled : isMaxReached()}
```

Comment:
fyi: you could do `increaseDisabled ?? isMaxReached()` (i often forget the existence of this as well).

Reply:
Changed it, thanks for reminding me!